lib/structuraid_core/engineering/analysis/footing/centric_combined_two_columns.rb

```diff
@@ -0,0 +1,85 @@
+module StructuraidCore
+  module Engineering
+    module Analysis
+      module Footing
+        class CentricCombinedTwoColumns < Base
+          include Utils::Data
+          include Utils::Centroid
+          include Utils::ShearMoment
+
+          def initialize(footing:, loads_from_columns:, section_direction:)
+            if ORTHOGONALITIES.none?(section_direction)
+              raise Engineering::Analysis::SectionDirectionError.new(section_direction, ORTHOGONALITIES)
+            end
+
+            @footing = footing
+            @loads_from_columns = loads_from_columns
+            @section_direction = section_direction
+          end
+
+          def build_geometry(coordinates_system)
+            @coordinates_system = coordinates_system
+
+            coordinates_system.relative_locations.clear
+            relativize_loads
+
+            aligner_vector = coordinates_system.relative_locations.first.to_vector
+            coordinates_system.align_axis_1_with(vector: aligner_vector)
+            locations_with_edges = include_edges_location
+
+            coordinates_system.relative_locations.clear
+            locations_with_edges.each { |location| coordinates_system.add_location(location) }
+          end
+
+          def reaction_1
+            -(solicitation * @footing.horizontal_area + reaction_2)
+          end
+
+          def reaction_2
+            (solicitation_load / 2 / long_stretch_2) * (long_stretch_1**2 - (long_stretch_2 + long_stretch_3)**2)
+          end
```

Comment:
what do `reaction_1` and `reaction_2` mean? Can they have a more descriptive name?

Reply:
Done!
- `reaction_at_first_column`
- `reaction_at_second_column`

Reply:
Done!
- `reaction_at_first_column`
- `reaction_at_second_column`

Reply:
cool!